### ISSUE TITLE
Package breaks highlighting on PHP/Blade files when installed alongside `laravel-blade` package.

### ISSUE THREAD

**Issue by ryami333** [y0ssar1an/CSS3#78]
When both are installed, and a php/blade file is opened, this error occurs:

```
Error loading syntax file "Packages/Laravel Blade Highlighter/blade.sublime-syntax": Apparent recursion within a with_prototype action: 25000 context sanity limit hit
```

Using build 3126 of Sublime. Removing either package fixes the issue.

**Comment by ryami333** [comment]
Not sure on which package the fault lies, so have opened the equivalent issue on the `laravel-blade` repo: https://github.com/Medalink/laravel-blade/issues/143.

**Comment by y0ssar1an** [comment]
Reposting my comments on [the Laravel Blade issue](https://github.com/Medalink/laravel-blade/issues/137#issuecomment-318932076) here.

>Hi, I'm the author of CSS3. I was able to reproduce the issue when both the default CSS package and CSS3 were enabled. I recommend disabling the default CSS package in my install notes, but I understand that many people can't do that because they use a package that depends on the default CSS package (like Laravel Blade).

>The clumsy workaround is to enable/disable CSS and CSS3. When you're working on blade files, you can Package Control: Disable Package CSS3 and enable CSS. If you're working primarily with pure CSS files, I recommend disabling CSS and enabling CSS3.

>Sorry, I know that's an awkward solution. If I think of something better, I'll post it here.

**Comment by ethanbeyer** [comment]
Has any progress on this bug been made?

**Comment by y0ssar1an** [comment]
I believe I've fixed this bug in #108. Please see the PR notes for details. I'm going to close this issue for now. Please reopen if you encounter the sanity limit error again. My fix just squeaks under the 25,000 context limit. It's possible some syntaxes might still exceed the limit, in which case I've got some more cutting to do.
